packages/colorpickers/src/elements/ColorPicker/index.tsx

```diff
@@ -0,0 +1,269 @@
+            </StyledHueField>
+            <StyledAlphaField>
+              <Label hidden>{labels.alphaSlider || 'Alpha slider'}</Label>
+              <StyledAlpha
+                max={1}
+                step={0.01}
+                value={computedColor.alpha / 100}
+                onChange={handleAlphaSliderChange}
+              />
+              <StyledCheckered />
+              <StyledAlphaGradient
+                red={computedColor.red}
+                green={computedColor.green}
+                blue={computedColor.blue}
+              />
+            </StyledAlphaField>
+          </StyledSliders>
+        </StyledSliderGroup>
+        <StyledInputGroup>
+          <StyledHexField>
+            <StyledLabel isRegular>{labels.hex || 'Hex'}</StyledLabel>
+            <StyledInput
+              isCompact
+              maxLength={7}
+              value={state.hexInput}
+              /* eslint-disable jsx-a11y/no-autofocus */
+              autoFocus={autofocus}
+              onBlur={handleBlur}
+              onChange={handleHexChange}
+            />
+          </StyledHexField>
+          <StyledRGBAField>
+            <StyledLabel isRegular>{labels.red || 'R'}</StyledLabel>
+            <StyledInput
+              isCompact
+              type="number"
+              min="0"
+              max="255"
+              maxLength={3}
+              value={state.redInput}
+              onBlur={handleBlur}
+              onChange={handleRedChange}
+            />
+          </StyledRGBAField>
+          <StyledRGBAField>
+            <StyledLabel isRegular>{labels.green || 'G'}</StyledLabel>
+            <StyledInput
+              isCompact
+              type="number"
+              min="0"
+              max="255"
+              maxLength={3}
+              value={state.greenInput}
+              onBlur={handleBlur}
+              onChange={handleGreenChange}
+            />
+          </StyledRGBAField>
+          <StyledRGBAField>
+            <StyledLabel isRegular>{labels.blue || 'B'}</StyledLabel>
+            <StyledInput
+              isCompact
+              type="number"
+              min="0"
+              max="255"
+              maxLength={3}
+              value={state.blueInput}
+              onBlur={handleBlur}
+              onChange={handleBlueChange}
+            />
+          </StyledRGBAField>
+          <StyledRGBAField>
+            <StyledLabel isRegular>{labels.alpha || 'A'}</StyledLabel>
+            <StyledInput
+              isCompact
+              type="number"
+              min="0"
+              max="100"
+              value={state.alphaInput}
+              onBlur={handleBlur}
+              onChange={handleAlphaChange}
+            />
+          </StyledRGBAField>
+        </StyledInputGroup>
+      </StyledColorPicker>
+    );
+  }
+);
+
+ColorPicker.defaultProps = {
+  defaultColor: '#ffffff'
```

Comment:
Prefer shorthand `#fff` as this is the value that the ColorPicker renders for white.